### DISCUSSION TITLE
Assincronia

### DISCUSSION THREAD

**Discussion by ThiagoBussola** [TI-UNICESUMAR/2024-topicos-em-programacao-ESOFT7S#1] [Node.js]
O paradigma assíncrono é uma parte crucial do desenvolvimento Javascript, entender suas peculiaridades irá auxiliar você a criar aplicar melhores práticas para desenvolver e performar suas aplicações.

Para auxiliar seus conhecimentos nesse tema, leia o capítulo 3 do livro: Node.js Design Patterns 3 edition.
Coloque neste tópico suas dúvidas, pontos que achou interessantes e materiais complementares que você utilizou para entender o tema (artigos, vídeos, podcasts, etc).

**Comment by LuigiCapoia** [comment]
https://github.com/LuigiCapoia/topicos-em-programacao
